sdk/storage/storage-file/src/FileClient.ts

```diff
@@ -1990,16 +1992,53 @@ export class FileClient extends StorageClient {
     }
   }
 
+  /**
+   * Force close all handles for a file.
+   * @see https://docs.microsoft.com/en-us/rest/api/storageservices/force-close-handles
+   *
+   * @param {FileForceCloseHandlesOptions} [options] Options to force close handles operation.
+   * @returns {Promise<number>}
+   * @memberof FileClient
+   */
+  public async forceCloseAllHandles(options: FileForceCloseHandlesOptions = {}): Promise<number> {
```

Comment:
Why not make it pageble like other segmement APIs?

An internal loop may cause "unresponsive" for large number of handles.

Reply:
In general we don't add paged async iterators for operations that do actions but not retrieving data.  /cc @bterlson